tcms/templates/registration/login.html

```diff
@@ -28,7 +28,7 @@
         </div>
         <div class="form-group">
           <div class="col-xs-8 col-sm-offset-2 col-sm-6 col-md-offset-2 col-md-6">
-            <span class="help-block">Forgot <a href="{% url "tcms-password_reset" %}" tabindex="6">password</a>?</span>
+              <span class="help-block">{% trans "Forgot" %} <a href="{% url "tcms-password_reset" %}" tabindex="6">{% trans "password" %}</a>?</span>
```

Comment:
These should be a single string surrounded by `blocktrans`. Same for the rest of the registration templates.